Modules/Quotas/src/Quotas.Infrastructure/Persistence/Repository/RelationshipTemplatesRepository.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Backbone.Modules.Quotas.Application.Infrastructure.Persistence.Repository;
+using Backbone.Modules.Quotas.Domain.Aggregates.Relationships;
+using Backbone.Modules.Quotas.Infrastructure.Persistence.Database;
+using Enmeshed.DevelopmentKit.Identity.ValueObjects;
+using Microsoft.EntityFrameworkCore;
+
+namespace Backbone.Modules.Quotas.Infrastructure.Persistence.Repository;
+internal class RelationshipTemplatesRepository : IRelationshipTemplatesRepository
+{
+    private readonly IQueryable<RelationshipTemplate> _templatesReadOnly;
```

Comment:
We typically name these variables with readOnly as a prefix and not a suffix (e.g., _readOnlyTemplates)